src/app/resolvers/tech-record-validate/tech-record-validate.resolver.ts

```diff
@@ -59,9 +68,6 @@ export const techRecordValidateResolver: ResolveFn<boolean> = () => {
     map(() => {
       return true;
     }),
-    catchError(() => {
```

Comment:
Can you check the rsjx docs for the errors store.select could throw as assumingly if the store breaks we would still need this ?

Reply:
So looking at the docs select doesn't throw an error, also we're not throwing an error ourselves. selectTechRecord is always returning either techRecord or editingTechRecord and techRecord is initially undefined. There's now a check for if there is no tech record we navigate back